packages/ember-auto-import/ts/auto-import.ts

```diff
@@ -115,11 +116,40 @@ export default class AutoImport implements AutoImportSharedAPI {
     // "dependencies" of the app even though they're not really.
     this.rootPackage.magicDeps = this.v2Addons;
 
+    /**
+     * ember-source is the only known trouble-maker here, so
+     * we gather ember-source's bundled dependencies, and prevent them
+     * from being included in webpack.
+     *
+     * As ember-source migrates to a Real Package™, this list will automatically update.
+     * However, it is dependent on ember-source keeping the same general folder structure.
+     * This is technically private API, so it's still not bullet proof.
+     */
+    let alreadyBundled = new Set();
+
+    emberSource: if (this.rootPackage.hasDependency('ember-source')) {
+      let emberPackageJsonPath = resolvePackagePath(
+        'ember-source',
+        this.rootPackage.root
+      );
+
+      if (!emberPackageJsonPath) break emberSource;
```

Comment:
Let's not use labeled break when if statements would do just as well.

Reply:
Updated, thanks!